pkg/sql/executor_ir.go

```diff
@@ -203,16 +215,21 @@ func runSQLProgram(wr *PipeWriter, sqlProgram string, db *database.DB, modelDir
 	//
 	// The IR generation on the second statement would fail since it requires inspection the schema of some_table,
 	// which depends on the execution of create table some_table as (select ...);.
+	cwd, err := ioutil.TempDir("/tmp", "sqlflow_models")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(cwd)
```

Comment:
Do all the statements in a SQL program share the same temporary dir?

Reply:
Yes.

Reply:
If there are more than one `train` statements in the program, may the first one interfere with the second one? Will the checkpoints of the 1st `train` remains in the `model_save` directory?

Reply:
I agree with @shendiaomo. Each train/pred SQL should have exactly one working directory, and it is separate from other SQL's working directory.

Reply:
Thanks for this information, @shendiaomo @tonyyang-svail, done updating.